client/src/utils/create-sendgrid-contact.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+// Sendgrid secrets
+const sendgrid_contacts_url = process.env.REACT_APP_SENDGRID_CONTACS_API_URL;
+const sendgri_api_key = process.env.REACT_APP_SENDGRID_API_KEY;
+const sendgrid_list_id = process.env.REACT_APP_SENDGRID_LIST_ID;
+
+let sendgridContactsObject = {
+  contacts: [
+    {
+      list_ids: [sendgrid_list_id],
+    },
+  ],
+  list_ids: [sendgrid_list_id],
+};
+
+const createSengridContact = async (stepsData) => {
+  Object.assign(sendgridContactsObject.contacts[0], stepsData);
+  try {
+    const res = await axios.put(sendgrid_contacts_url, sendgridContactsObject, {
```

Comment:
Since I am not using the response here, perhaps I can just have the axios request but we will see if I log something or do something with the response (it returns a pretty useless job id)...